README.md

```diff
@@ -111,14 +111,14 @@ const axios = require('axios');
 
 // Make a request for a user with a given ID
 axios.get('/user?ID=12345')
-  .then(function (response) {
-    // handle success
-    console.log(response);
-  })
   .catch(function (error) {
     // handle error
     console.log(error);
   })
+  .then(function (response) {
+    // handle success
+    console.log(response);
+  })
   .then(function () {
     // always executed
   });
```

Comment:
I can understand the purpose to guide users how to catch errors caused by axios only. But it looks strange `catch..then`, especially here with 2 continuous `then`.

On the other hand, axios is a Promise based client, which will not pay too much attentions to docs about chained promise calls. axios focuses on sending requests and handling responses.

Reply:
Actually we can talk about two styles: `request.then(success, error)` or `request.catch(erro).then(success)`. The current way is wrong, It can't be used on a real software.

I understand that it's not part of axios code, but since axios is a very important and relevant tool in our community, that doc sometimes act as a reference, mainly for newbies.

Reply:
Maybe we can mention it by an additional sentence, instead of changing all examples.